Jenkinsfile

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env groovy
+
+/**
+ * Build Contiki Applications
+ *
+ */
+
+properties([
+    buildDiscarder(logRotator(numToKeepStr: '5')),
```

Comment:
This project is tiny, I suggest you bump this up to something like 30 as we are finding 5 not enough meaning builds get cleared away before you have a chance use them.